### PR TITLE
Updates cloudversion and images list for CC/3.15 release.

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

File: calico-cloud/variables.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const variables = {
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
   cloudversion: 'v3.14.1-16',
```
Comment (jaderhs) [inline calico-cloud/variables.js:15]: Does the `cloudversion` needs to be updated here to `v3.15.1-0`?

File: calico-cloud_versioned_docs/version-3.15/variables.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ const variables = {
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.14.1-16',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.14.1-16',
+  cloudversion: 'v3.15.1-0',
+  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.15.1-0',
```
Comment (jaderhs) [inline calico-cloud_versioned_docs/version-3.15/variables.js:16]: Does the `releaseTitle` variable needs to be updated here to `v3.15.1-0`?

**Review by jaderhs — APPROVED** [review]

**Comment by jaderhs** [comment]
LGTM

File: calico-cloud_versioned_docs/version-3.15/variables.js (hunk 2 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/variables.js:16]: That's actually a relic from the other product docs. We don't use releaseTitle in CC.

Plans in the works for cleaning up variables.

**Pull request merged**